pkg/provider/aws/action/rhel/cloud-config-snc

```diff
@@ -5,26 +5,24 @@ rh_subscription:
   auto-attach: true
 runcmd:
   - while fuser /var/lib/rpm/.rpm.lock > /dev/null 2>&1 ; do sleep 1 ; done
+  - dnf clean all
   - dnf install -y podman qemu-kvm libvirt virt-install virt-viewer jq git
   # Debug libvirt
   #- echo 'log_filters="1:libvirt 1:util 1:qemu"' | tee -a /etc/libvirt/libvirtd.conf
   #- echo 'log_outputs="1:file:/var/log/libvirt/libvirtd.log"' | tee -a /etc/libvirt/libvirtd.conf
   # https://libvirt.org/manpages/libvirtd.html#system-socket-activation
   - echo 'LIBVIRTD_ARGS="--listen"' | tee -a /etc/sysconfig/libvirtd
-  - echo 'listen_tls = 0' | tee -a /etc/libvirt/libvirtd.conf
-  - echo 'listen_tcp = 1' | tee -a /etc/libvirt/libvirtd.conf
-  - echo 'tcp_port = "16509"' | tee -a /etc/libvirt/libvirtd.conf
-  - echo 'auth_tcp = "none"' | tee -a /etc/libvirt/libvirtd.conf
   - systemctl daemon-reload 
   - systemctl enable libvirtd-tcp.socket 
-  - systemctl start --no-block libvirtd-tcp.socket 
+  - systemctl start libvirtd-tcp.socket 
   - systemctl mask libvirtd.socket libvirtd-ro.socket libvirtd-admin.socket libvirtd-tls.socket libvirtd-tcp.socket
   - systemctl enable libvirtd 
-  - systemctl start --no-block libvirtd  
+  - systemctl start libvirtd  
```

Comment:
in the rhel9 docs there's oneline to start all the libvirt related sockets services: `for drv in qemu network nodedev nwfilter secret storage interface; do systemctl start virt${drv}d{,-ro,-admin}.socket; done`

from: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/assembly_enabling-virtualization-in-rhel-9_configuring-and-managing-virtualization#proc_enabling-virtualization-in-rhel-9_assembly_enabling-virtualization-in-rhel-9

Reply:
Ok, I am gonna give try, but I only will include this if it is supported on RHEL 8 as well, as the current cmd works on both and I do not wanna add complexity with diff code based on RHEL version (unless it is totally necesary)

Reply:
Tested with both RHEL `9.4` and `8.10` working with both